src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -11,6 +11,13 @@
      * Return false in any other cases.
      */
     public boolean login(String email, String password) {
-        return false;
+        UserService checkUser = new UserService();
+        if (checkUser.findByEmail(email) != null
+                && checkUser.findByEmail(email).getEmail().equals(email)
+                && checkUser.findByEmail(email).getPassword().equals(password)) {
```

Comment:
Should we search the user inside the database three times?

Reply:
I fixed this problem.